.github/workflows/check_examples.yaml

```diff
@@ -31,6 +31,8 @@ jobs:
         with:
           go-version: ${{ steps.go-version.outputs.content }}
       - uses: engineerd/setup-kind@v0.5.0
+        with:
+          version: "v0.10.0"
```

Comment:
I'm confused about these version statements... I know it matches the Kind docs to specify both `@v0.5.0` and `version: "v0.10.0" together... but I'm not sure what that means. It seems to contradict the Github Workflow docs, which say that the `@` symbol is what specifies the version. https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions 

ohh! I see now. It took a little investigating, but v0.5.0 is the action version, and v0.10.0 is the Kind version. This runs Kubernetes API version 1.20, according to the release docs.

https://github.com/kubernetes-sigs/kind/releases/tag/v0.10.0

But still, it could be nicer to just toss a `kubectl version` or something similar into the run action, to give us an  "at a glance" view of the API version. Just a suggestion.

Reply:
Yeah, your conclusion about which version is what is correct.
The setup-kind action actually supports setting explicit configuration to be passed to `kind` and that includes thee option to set the exact cluster version it will bring up.
I think we should consider doing that. 

But I also like the idea of reporting the version on the client side. Could we maybe do that in the test code itself? Say in TestMain when we build a client-go instance? Use client-go to check and report API version? That way we avoid the need for kubectl in the runner environment and we also get the version reported no matter where the tests run.

Reply:
Sure, I'd be happy with anything that reports the API version. Though, just FYI, according to the Kind docs, the github workers come with kubectl already configured. They show using kubectl with their github action on this page: https://github.com/engineerd/setup-kind

But if you prefer it in the test code, that's ok too.

Reply:
I added version logging from `TestMain()`. It looks like this:
```
go test -count=1 -tags acceptance "./test/acceptance" -v  -timeout 120m
2021/04/16 22:59:33 Testing against Kubernetes API version: v1.20.2
```